config.js

```diff
@@ -58,7 +53,7 @@ const initConfig = callback => {
       config.schedule_evening_hours,
       config.schedule_evening_minutes
     );
-    callback();
+    require('./transitions').loadTransitions();
```

Comment:
Is there a reason we require this every time btw? AFAICT require doesn't do anything after the first go:
```js
> require('./requireme')
You loaded me!
{}
> require('./requireme')
{}
```

So it really just seems to be a shortcut if you don't want to create a variable to store the reference in.